ts/components/screens/ScreenContentHeader.tsx

```diff
@@ -20,8 +20,9 @@ import { Body } from "../core/typography/Body";
 
 type Props = Readonly<{
   title?: string;
-  icon?: ImageSourcePropType;
-  iconFont?: IconProps;
+  rasterIcon?: ImageSourcePropType;
+  icon?: IOIcons;
+  pictogram?: IOSectionPictogramType;
   subtitle?: string;
   subtitleLink?: JSX.Element;
```

Comment:
same here for the class syntax, I think we should list all of these "more legacy" files in a task in order to give them a touchup